router/logging.go

```diff
@@ -103,9 +104,18 @@ func (l *LoggingMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	l.Next.ServeHTTP(rlogger, r)
 
 	log.Debugln("------ Response: ------")
-	if !skipBody && (w.Header().Get("Content-Type") == "" || w.Header().Get("Content-Type") == "application/json") {
+	if !skipBody && isBodyLoggable(r.Header) {
 		log.Debugln(rlogger.String())
 	} else {
 		log.Debugf("%d bytes of response body", len(rlogger.String()))
 	}
 }
+
+func isBodyLoggable(h http.Header) bool {
+	contentType := h.Get("Content-Type")
+	mediaType, _, err := mime.ParseMediaType(contentType)
+	if err != nil {
+		return false
+	}
+	return mediaType == "" || mediaType == "application/json"
+}
```

Comment:
For logic that only usable to `LoggingMiddleware` I would prefer you put it on struct.

And it will be great this logger can use use as following:

```
finalMux = &router.LoggingMiddleware{
	Skips: []string{
		"/files/",
		"/_/pubsub/",
		"/pubsub/",
	},
	MimeConcern: []string{
		"application/json"
	},
	Next: finalMux,
}
```